.nvmrc

```diff
@@ -1 +1 @@
-v10.16.0
+v10.15.3
```

Comment:
You should PSA in group chat before merging this. I am not sure how this will unfold for folks who may not have this version installed. The fix is presumably easy, running some nvm command, do you know what it is? Should tell folks.

Reply:
It should be `nvm use`, it defaults to look for the nvmrc.